superset-frontend/src/components/Modal/Modal.tsx

```diff
@@ -106,30 +111,41 @@ export const StyledModal = styled(BaseModal)<StyledModalProps>`
       top: 0;
     `}
 
-  .ant-modal-content {
+  .antd5-modal-content {
     display: flex;
     flex-direction: column;
     max-height: ${({ theme }) => `calc(100vh - ${theme.gridUnit * 8}px)`};
     margin-bottom: ${({ theme }) => theme.gridUnit * 4}px;
     margin-top: ${({ theme }) => theme.gridUnit * 4}px;
+    padding: 0;
   }
 
-  .ant-modal-header {
+  .antd5-modal-header {
     flex: 0 0 auto;
-    background-color: ${({ theme }) => theme.colors.grayscale.light4};
     border-radius: ${({ theme }) => theme.borderRadius}px
```

Comment:
Can this be centralized / removed / or moved to theme level?

Reply:
This is already centralized as it is configured in the main Modal component